flexget/plugins/input/anilist.py

```diff
@@ -122,54 +122,65 @@ def on_task_input(self, task, config):
             try:
                 list_response = list_response.json()['data']
                 logger.debug('JSON output: {}', list_response)
-                for list_status in list_response['collection']['statuses']:
+                for list_status in list_response.get('collection', {}).get('statuses', []):
                     if (
                         selected_list_name
-                        and list_status['name'].lower() not in selected_list_name
+                        and list_status.get('name', 'None').lower() not in selected_list_name
                     ):
                         continue
                     for anime in list_status['list']:
-                        anime = anime['anime']
+                        anime = anime.get('anime')
                         has_selected_release_status = (
-                            anime['status'].lower() in selected_release_status
+                            anime.get('status', 'None').lower() in selected_release_status
```

Comment:
My comment was on an outdated line-- I don't like `'None'` because it could be a valid value. If you absolutely need a value, I'd use an empty string.